<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPrefProductosBase</class>
 <widget class="QWidget" name="FormPrefProductosBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Elementos a mostrar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="CkBDescripcion">
        <property name="text">
         <string>Mostrar Descripcion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBMarcas">
        <property name="text">
         <string>Utilizar Marcas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBCategorias">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Utilizar Categorias</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBCosto">
        <property name="text">
         <string>Mostrar precio de costo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBStock">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Habilitar Control de Stock</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="CkBPermitir">
        <property name="text">
         <string>Permitir seguir vendiendo aunque no exista stock disponible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBLimitarVenta">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Limitar venta posible al stock actual del producto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBAvisosStock">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Habilitar avisos de stock bajo</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LStockMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Limite de stock minimo:</string>
          </property>
          <property name="buddy">
           <cstring>DSBLimiteMinimo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="DSBLimiteMinimo">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBGanancia">
     <property name="title">
      <string>Ganancia Predeterminada</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Porcentaje de Ganancia:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="DsBGanancia">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CkBAvisosStock</sender>
   <signal>toggled(bool)</signal>
   <receiver>LStockMin</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CkBAvisosStock</sender>
   <signal>toggled(bool)</signal>
   <receiver>DSBLimiteMinimo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
